keylime-ansible-gcp/roles/create-gcp/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
If we don't have any handlers, then we can just omit this file. It's not necessary to have a blank file/directory in the repo and if anyone ever needs it in the future they can always add it.